app/views/bookmarks/list.html.erb

```diff
@@ -83,6 +84,7 @@
           <% end %>
         </td>
         <td>
+          <!--Users can only add/edit/view review other users' bookmarks.-->
```

Comment:
I need to fix this. Should say "Users can only review bookmarks that they did not add, and can only view/edit their own reviews."